controllers/bootstrap/init.go

```diff
@@ -1180,8 +1180,19 @@ func (b *Bootstrap) DeployCertManagerCR() error {
 // NamespaceScope resources include common-service, nss-managedby-odlm, nss-odlm-scope, and odlm-scope-managedby-odlm
 func (b *Bootstrap) CleanNamespaceScopeResources() error {
 
+	// get namespace-scope ConfigMap in operatorNamespace
+	nssCmNs, err := util.GetNssCmNs(b.Reader, b.CSData.OperatorNs)
+	if err != nil {
+		klog.Errorf("Failed to get %s configmap: %v", constant.NamespaceScopeConfigmapName, err)
+		return err
+	} else if nssCmNs == nil && err == nil {
```

Comment:
I believe we do not need to check `err == nil`, it has been checked by `err != nil` already.
```suggestion
	} else if nssCmNs == nil {
```